### PR TITLE
Add support of 'rebond' and 'withdraw_unbonded' tx types

### PR DESCRIPTION
**Pull request by haammar-ledger**
- Add support of 'rebond' and 'withdraw_unbonded' tx types from CLI
- Add tests & error cases for getTransactionStatus
- Try to add consistency for operation types for Polkadot: use new BOND, UNBOND and REBOND types, but use existing DELEGATE and UNDELEGATE for nominate and chill tx types (still WIP, can be discussed).

### CODE REVIEW

File: src/types/operation.js [hunk 1]
```diff
@@ -29,9 +29,7 @@ export type OperationType =
   | "APPROVE"
   | "OPT_IN"
   | "OPT_OUT"
-  | "CLOSE_ACCOUNT"
-  | "SUPPLY"
-  | "REDEEM";
```
Comment (henri-ly) [inline src/types/operation.js:34]: I don't think this should be delete

File: src/families/polkadot/js-getTransactionStatus.js [hunk 2]
```diff
@@ -110,52 +111,77 @@ const getTransactionStatus = async (a: Account, t: Transaction) => {
...
+
+  const currentBonded = a.polkadotResources?.bondedBalance.minus(currentUnbonding);
+
   switch (t.mode) {
     case "bond":
       // Not a stash yet -> bond method sets the controller
       if (!isStash(a)) {
         if (!t.recipient) {
           errors.recipient = new RecipientRequired("");
+          break;
```
Comment (rjarasson-ledger) [inline src/families/polkadot/js-getTransactionStatus.js:134]: Pas nécessaire de break, on est en else-if, et il est préférable d'avoir plusieurs erreurs (donc de check aussi l'amount)

File: src/types/operation.js [hunk 3]
```diff
@@ -29,9 +29,7 @@ export type OperationType =
   | "APPROVE"
   | "OPT_IN"
   | "OPT_OUT"
-  | "CLOSE_ACCOUNT"
-  | "SUPPLY"
```
Comment (rjarasson-ledger) [inline src/types/operation.js:33]: Euh... j'y toucherais pas perso aux types de COMPOUND...

**Review by rjarasson-ledger — CHANGES_REQUESTED** [review]

File: src/types/operation.js (hunk 1 above)
Comment (haammar-ledger) [reply src/types/operation.js:34]: SUPPLY and REDEEM are already defined L27 & L28, no?

**Review by rjarasson-ledger — APPROVED** [review]

**Pull request merged**